_posts/2020-06-20-Basic-Querying.md

```diff
@@ -0,0 +1,302 @@
+
+This is a difficult passage and the interpretation above is provisional. In particular, the genitive singular of Irish _[ben](http://www.dil.ie/5644)_ should be _mná_, not _*ba[i]ne_. The genealogies in the [Book of Lecan](https://www.vanhamel.nl/codecs/Dublin,_Royal_Irish_Academy,_MS_23_P_2) (not yet part of IrishGen) also contain this passage and read "Diabalmuine" ("of the double gift"?; see eDIL, _s.v._ _[díabul](http://www.dil.ie/15864)_).
+
+In any case, we've shown how a query can retrieve information about an individual and about individuals connected to them in some way from across IrishGen's database. Obviously this can only be done with material that has already been added to IrishGen and which we have encoded correctly, including correctly predicating which mentions of people relate to the same individuals. We are thus searching material with which we've already worked quite closely. However, querying is invaluable even for us, as IrishGen already contains almost 25,000 individuals and we have been working on it for several years, so we need to be able to keep track of our data.
+
+## Searching for the Satirist
+Returning to Text 1, some intriguing information is provided about the fortunes and progeny of one of Findchán's sons (**[R §1347](https://celt.ucc.ie/published/G105003/text018.html)**):
+
+> Coarthach mc-side Loiscibet .i. bancháinte do Chiannacht Glinni Gemen do Cheníul Fergusa m. Lemna m. Cóelbad do-bert tír dó conid de at-berar Coartrige.
+> > Coarthach was the son of Loiscibet, that is, the female satirist of the Cíannachta Glinne Geimen, of the kindred of Fergus, son of Liaman, son of Cóelbad, who gave land to him, so that from him the Coartraige are named.
+
+I originally planned to make this post about identifying Loiscibet and her ancestors by querying IrishGen's data. However, extensive querying of IrishGen and consultation of other sources has failed to turn up any convincing identifications. This might be illustrative of what the frustrations that can occur while working with this sort of source, but it wouldn't be very useful for demonstrating querying.
+
+What we can do is use IrishGen to contextualise Loiscibet in terms other than of ancestry. Her entry relating to her appearance in the above passage is as follows:
+
+```
+<http://example.com/Rawl_B502/clanna_ébeir_h_i_l_leith_chuind.trig#Loiscibet>
+    a foaf:Person ;
+    irishRel:genName "Loiscibet" ;
+    foaf:gender "female" ;
+    foaf:title "bancháinte"@sga, "female satirist"@eng;
+    owl:sameAs <http://example.com/LL/clanna_ébir_i_l-leith_chuind.trig#Loscibet>;
+    rdfs:comment "bancháinte do Chiannacht Glinni Gemen" ;
+    rel:parentOf <#Coarthach>.
+```
+
+She is described as a _bancháinte_ ("female satirist"), from _ben(http://www.dil.ie/5644)_ ("woman") and _[cáinte](http://www.dil.ie/7854)_ ("satirist"). Do any other satirists appear in IrishGen's database? We can find out:
```

Comment:
_ben(http://www.dil.ie/5644)_ needs to be fixed as the link is not rendering correctly.  It probably should be _[ben](http://www.dil.ie/5644)_